app/mailers/jobapps_mailer.rb

```diff
@@ -42,6 +42,16 @@ def interview_reschedule(interview)
          reply_to: reply_to
   end
 
+  def send_note_for_later(application_record)
```

Comment:
@akaplo I agree that's a good thought - feel free to open an issue about it referencing this PR.